pkg/reconciler/revision/table_test.go

```diff
@@ -623,6 +640,13 @@ func deployImagePullSecrets(deploy *appsv1.Deployment, secretName string) *appsv
 	return deploy
 }
 
+func deployNodeSelector(deploy *appsv1.Deployment, nodeSelectorKey, nodeSelectorValue string) *appsv1.Deployment {
```

Comment:
I'd expect changes to the `deploy_test.go` as well.

Reply:
Done.